ee/src/main/java/org/jboss/as/ee/subsystem/ManagedThreadFactoryRemove.java

```diff
@@ -35,9 +33,4 @@
     private ManagedThreadFactoryRemove() {
         super(ManagedThreadFactoryAdd.INSTANCE);
     }
-
-    @Override
-    protected ServiceName serviceName(String name) {
```

Comment:
Given that you are no longer overriding behavior of ServiceRemoveStepHandler, we can eliminate this class altogether.